### DISCUSSION TITLE
Convert motion photo to video/GIF?

### DISCUSSION THREAD

**Discussion by lbdunk** [deckerst/aves#1250] [Q&A]
Hi! I had a quick question - is there a way to convert a motion photo to a video or GIF? When I select Convert, the format options available are BMP, JPEG, PNG, and WEBP. Thanks!
![Image20241018124224](https://github.com/user-attachments/assets/4203937b-e695-442d-a752-57583bf228db)
![Image20241018124228](https://github.com/user-attachments/assets/07c44690-f73d-45ec-be6f-06d7ef59f4a5)

**Comment by lbdunk** [comment]
Saw this is already listed as an issue!

**Comment by deckerst** [comment]
Not what you asked, but related actions:
- in the full screen viewer, when you long press the share button, you can choose to share only the photo, or only the video,
- in the Info page, in the top right menu, there's `Open video`, and then you could copy/move/share it.
